sdk/avs/Azure.ResourceManager.Avs/src/Generated/WorkloadNetworkResource.cs

```diff
@@ -27,10 +26,9 @@ public partial class WorkloadNetworkResource : ArmResource
         /// <param name="subscriptionId"> The subscriptionId. </param>
         /// <param name="resourceGroupName"> The resourceGroupName. </param>
         /// <param name="privateCloudName"> The privateCloudName. </param>
-        /// <param name="workloadNetworkName"> The workloadNetworkName. </param>
-        public static ResourceIdentifier CreateResourceIdentifier(string subscriptionId, string resourceGroupName, string privateCloudName, WorkloadNetworkName workloadNetworkName)
+        public static ResourceIdentifier CreateResourceIdentifier(string subscriptionId, string resourceGroupName, string privateCloudName)
```

Comment:
So in the new api-version, this resource becomes a singleton resource, therefore we see its collection removed.
Is this the case?

Reply:
Correct this is the case